recipes/slamdunk/meta.yaml

```diff
@@ -42,6 +42,7 @@ requirements:
     - curl >7.59
     - xz
     - libdeflate
+    - openssl=1.0
```

Comment:
Please remove the openssl pinning, as well as the version restriction for `curl`. Both of these should be coming from conda-forge-pinning.

Reply:
@dpryan79 The `curl` pinning was originally introduced by @bgruening to fix the latest build. So you are telling me removing the `curl` and `openssl` pinning and creating a new build number should fix all issues? The current build is definitely not working for people due to the newer `openssl` version.

Reply:
Yes, remove the pinning and just bump the build number. That should resolve the issue.